modules/installation-vsphere-steal-clock-accounting.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+// * installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vmc/installing-restricted-networks-vmc.adoc
+// * installing/installing_vmc/installing-vmc-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+
+:_content-type: CONCEPT
+[id="installation-vsphere-steal-clock-accounting_{context}"]
+= Steal clock accounting
+
+By default, the installation program provisions the cluster's virtual machines without enabling the steal clock accounting parameter (`stealclock.enabled`). Enabling steal clock accounting can help with troubleshooting cluster issues. After the cluster is deployed, use the vSphere Client to enable this parameter on each of the virtual machines.
```

Comment:
I don't believe `Client` needs to be capitalized here

Reply:
Hey @kelbrown20 `vSphere Client` is how VMware refers to the client.

Reply:
Gotcha! I didn't see it capitalized much used in the other docs which Is why I wasn't sure. But that makes sense 👍